config.toml

```diff
@@ -369,6 +369,10 @@ logo = "baseten.png"
 url = "https://www.bolddesk.com/"
 logo = "bolddesk.svg"
 
+[[params.users]]
```

Comment:
Welcome!

Could you please sort this alphabetically.

Reply:
Sure @zroubalik can you check now?